gluon/contrib/pg8000/__init__.py

```diff
@@ -1,3 +1,14 @@
+from .core import (
+    Warning, Bytea, DataError, DatabaseError, InterfaceError, ProgrammingError,
+    Error, OperationalError, IntegrityError, InternalError, NotSupportedError,
+    ArrayContentNotHomogenousError, ArrayContentEmptyError,
+    ArrayDimensionsNotConsistentError, ArrayContentNotSupportedError, utc,
+    Connection, Cursor, Binary, Date, DateFromTicks, Time, TimeFromTicks,
+    Timestamp, TimestampFromTicks, BINARY, Interval)
```

Comment:
thank the author of the module. My personal preference is still the usual multiple lines "from foo import x, y, z"

BTW: make a note to yourself... the pg8000 driver needs a quite light rewrite of imports to make it work under web2py env...